Documentation/project-docs/developer-guide.md

```diff
@@ -171,6 +182,23 @@ There may be multiple projects in some directories so you may need to specify th
 
 Tests participate in the incremental build.  This means that if tests have already been run, and inputs to the incremental build have not changed, rerunning the tests target will not execute the test runner again.  To force re-executing tests in this situation, use `msbuild /t:RebuildAndTest`.
 
+#### Running tests in a different TFM
+
+Each test project corresponds to a test .builds file. There are some tests that might be OS specific, or might be testing API that is available only on some TFMs, which is what this tests.builds files are for. By default, we will build all of these different configurations always, but we will only execute the tests on one TFM. By default, our `TestTFM` is `netcoreapp1.1` which means that we will run tests for the configurations that either:
+- Have `netcoreapp` inside their `<TestTFMs>...</TestTFMs>` clause on the .builds files, or
+- Don't have a `<TestTFMs>...</TestTFMs>` metadata in the .builds file at all.
```

Comment:
I would add because we default it to netcoreapp1.1 and add a pointer to where we default it.